slsDetectorServers/slsDetectorServer/include/slsDetectorFunctionList.h

```diff
@@ -25,7 +25,7 @@
 #include "blackfin.h"
 #endif
 
-#if defined(MYTHEN3D) 
+#if defined(MYTHEN3D)
```

Comment:
do we do #ifdef or if defined? See line 320 =)

Reply:
done